source/_seed_and_dew.html.erb

```diff
@@ -1,17 +0,0 @@
-<script>
-  var SeedAndDewConfig = {};
-  (function() {
-    SeedAndDewConfig['adClass'] = "snd-ad";
-    /* * * DON'T EDIT BELOW THIS LINE * * */
-    SeedAndDewConfig['projectId'] = 'ef35b4ec-231e-4950-bd93-4c06fe3b2543';
-    SeedAndDewConfig['loadStartTime'] = performance.now();
-    SeedAndDewConfig['apiVersion'] = '2018-05-28'
-    SeedAndDewConfig['sessionId'] = Math.random().toString(36).substring(2, 15);
-    var snd = document.createElement('script');
-    snd.type = 'text/javascript';
-    snd.async = true;
-    snd.src = 'https://www.seedanddew.com/static/embed.min.js';
-    (document.getElementsByTagName('head')[0] ||
-    document.getElementsByTagName('body')[0]).appendChild(snd);
-  })();
-</script>
```

Comment:
If we wish to stop our participation in seed and dew thats a discussion we can have, but whilst we're still setup with them and open collective receives money from them on our behalf...

Reply:
Oh, my bad. I got confused by this line in the console:
```
Loading failed for the <script> with source “https://www.seedanddew.com/static/embed.min.js”.
```

Might be my adblocker.
Where do those support buttons are supposed to appear? Can't see anything even in Incognito.